pkg/controller/migcluster/validation.go

```diff
@@ -39,13 +44,14 @@ const (
 
 // Messages
 const (
-	ReadyMessage              = "The cluster is ready."
-	MissingURLMessage         = "The `url` is required when `isHostCluster` is false."
-	InvalidSaSecretRefMessage = "The `serviceAccountSecretRef` must reference a `secret`."
-	InvalidSaTokenMessage     = "The `saToken` not found in `serviceAccountSecretRef` secret."
-	TestConnectFailedMessage  = "Test connect failed: %s"
-	MalformedURLMessage       = "The `url` is malformed."
-	InvalidURLSchemeMessage   = "The `url` scheme must be 'http' or 'https'."
+	ReadyMessage               = "The cluster is ready."
+	MissingURLMessage          = "The `url` is required when `isHostCluster` is false."
+	InvalidSaSecretRefMessage  = "The `serviceAccountSecretRef` must reference a `secret`."
+	InvalidSaTokenMessage      = "The `saToken` not found in `serviceAccountSecretRef` secret."
+	TestConnectFailedMessage   = "Test connect failed: %s"
+	MalformedURLMessage        = "The `url` is malformed."
+	InvalidURLSchemeMessage    = "The `url` scheme must be 'http' or 'https'."
+	UnauthorizedSaTokenMessage = "The service account token is not authorized to perform migrations."
```

Comment:
Saying that the token is _unauthorized_ may be misunderstood to mean that the token is invalid.

Suggest other naming/wording:
```
SaTokenNotPrivileged = The `saToken` has insufficient privileges.
```
To be consistent with other messages, we reference the specific field in ``.  The _"to perform migrations"_ is to leave but seems unnecessarily.